src/pages/label/Label.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import Navbar from '../../components/navbar/Navbar';
+import Sidebar from '../../components/sidebar/Sidebar';
+import { MdLabel } from "react-icons/md" ;
+
+function Label() {
+  return <>
+  <Navbar/>
```

Comment:
can keep `Navbar` out of routes, instead of importing in each component

Reply:
yeah i also thought the same will try to  implement